lntest/itest/lnd_test.go

```diff
@@ -203,6 +203,7 @@ func TestLightningNetworkDaemon(t *testing.T) {
 	// TODO(roasbeef): create master balanced channel with all the monies?
 	aliceBobArgs := []string{
 		"--default-remote-max-htlcs=483",
+		"--dust-threshold=5000000",
```

Comment:
Reminder to drop before merge

Reply:
dropped commit but kept the change since o/w a flake occurs in `async_bidi_payments`